ucp/useraccounts_module.php

```diff
@@ -484,9 +449,9 @@ function main($id, $mode)
 							AND item_id <> $item_id
 							AND user_id = $this_user_id";
 						$this->db->sql_query($sql);
-						$move_executed = (bool) $this->db->sql_affectedrows();
+						$move_executed = (bool)$this->db->sql_affectedrows();
```

Comment:
This cast isn't needed.

Reply:
It seems this needs to be a boolean expression for the json response to the ajax callback that triggers the reordering on the page or the js does not work.

```php
	// Response for the ajax callback that reorders items on the page.
	if ($this->request->is_ajax())
	{
		$json_response = new \phpbb\json_response;
		$json_response->send(array(
			'success' => $move_executed,
		));
	}
```

```js
phpbb.addAjaxCallback('row_down', function(res) {
	if (typeof res.success === 'undefined' || !res.success) {
		return;
	}
```